frontend/src/components/organisms/Header/index.jsx

```diff
@@ -43,7 +44,7 @@ const Header = () => {
       <MastheadContent>
         CPT Dashboard
         <div className="last-updated-box">
-          <div>Last Updated: 21/02/2024 13:26:30</div>
+          {updatedTime && <div>Last Updated: {updatedTime}</div>}
```

Comment:
That's interesting. Somehow I always figured that was supposed to be the last time the *dashboard implementation* was updated, and I wondered how the timestamp was set but never got around to looking. So it was just a hardcoded date, and now it's the last time the contents of the current tab were updated?

This doesn't seem all that useful ... although a hardcoded date doesn't help anyone much, either. I suspect we should have a product version that gets displayed on the masthead -- maybe for both the "dashboard" and the "aggregator" (backend) as we move forward, which would be presumably latched into redux from an API call on initial load.

For now ... I'm just kinda curious what the thought process was here?